ores/scoring_systems/celery_queue.py

```diff
@@ -0,0 +1,287 @@
+import logging
+import re
+from urllib.parse import urlparse
+from itertools import chain
+
+import celery
+import celery.exceptions
+import mwapi.errors
+import revscoring.errors
+from celery.signals import before_task_publish
+
+from ..errors import TimeoutError, ScoreProcessorOverloaded
+from .scoring_system import ScoringSystem
+
+logger = logging.getLogger(__name__)
+
+APPLICATIONS = []
```

Comment:
All caps generally denotes a static resource; this seems to not be the case here